### PR TITLE
Update and rename isaacgym-installation-troubleshooting.md to isaacgym-installation.md

### PR DESCRIPTION
**Pull request by GiulioRomualdi**
This adds the documentation that can be followed to install IsaacGym

### CODE REVIEW

**Review by flferretti — APPROVED** [review]

File: doc/isaacgym-installation.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+### ImportError: libpython
+
+```console
+ImportError: libpython3.7m.so.1.0: cannot open shared object file: No such file or directory
+```
+
+In your conda env run:
+
+```sh
+conda env config vars set LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/home/$(whoami)/mambaforge/envs/rlgpu/lib
```
Comment (traversaro) [inline doc/isaacgym-installation.md:123]: We can use ${CONDA_PREFIX} here to avoid the hardcoding.

**Review by traversaro — APPROVED** [review]
Minor comment.

File: doc/isaacgym-installation.md (hunk 1 above)
Comment (flferretti) [reply doc/isaacgym-installation.md]: Thanks Silvio! Addressed in https://github.com/ami-iit/ami-commons/pull/16/commits/ed057fc85503fb7024b08f29419d8a8ef18a9c6c

File: doc/isaacgym-installation.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+### ImportError: libpython
+
+```console
+ImportError: libpython3.7m.so.1.0: cannot open shared object file: No such file or directory
+```
+
+In your conda env run:
+
+```sh
+conda env config vars set LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$CONDA_PREFIX
```
Comment (traversaro) [inline doc/isaacgym-installation.md:123]: In this version we are missing the `/lib` directory:
```suggestion
conda env config vars set LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$CONDA_PREFIX/lib
```

File: doc/isaacgym-installation.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+```
+where `mambaforge` should be substituted with `anaconda3` if you are using Anaconda.
```
Comment (traversaro) [inline doc/isaacgym-installation.md:125]: If use use `$CONDA_PREFIX`, we can get rid of this sentence.

Comment (traversaro) [inline doc/isaacgym-installation.md:125]:
```suggestion
```

If use use `$CONDA_PREFIX`, we can get rid of this sentence.

Comment (flferretti) [reply doc/isaacgym-installation.md]: You're right, addressed in a79c92ff9539051016bd5839c6d99d6cfa2b1d3f

**Pull request merged**